public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -149,25 +150,34 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
             <ToolbarButton
               variant="canvas"
               key="split"
-              tooltip="Split the pane"
+              tooltip={t('explore.toolbar.split.tooltip', 'Split the pane')}
```

Comment:
I think we should stick with 3-level IDs (naming [convention](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#phrase-id-naming-convention)). How about `explore.toolbar.split-tooltip`?

Reply:
Thank you for your feedback. addressed it here.

```suggestion
              tooltip={t('explore.toolbar.split-tooltip', 'Split the pane')}
```